.github/ISSUE_TEMPLATE/issue_report.yml

```diff
@@ -0,0 +1,62 @@
+name: Issue report
```

Comment:
it should be named `.md`

Reply:
@kkyusufk you mean to say that , name should be like this  `name : Issue.md` or i have to change `.yml` to `.md`

Reply:
`issue.md`

Reply:
changes done

Reply:
Yeah this is my bad, I will revert the push which I did.
I was a bit confused.